ruby_event_store_rom_sql/lib/ruby_event_store/expected_version.rb

```diff
@@ -0,0 +1,57 @@
+module RubyEventStore
+  class ExpectedVersion
+    POSITION_DEFAULT = -1.freeze
```

Comment:
Minor but I believe integers are already frozen:

```
irb(main):001:0> -1.frozen?
=> true
```

Reply:
@pawelpacana I want to say that the mutant yelled at me until I did this.